benchmarks/dynamo/microbenchmarks/inductor_cpu_atomic.py

```diff
@@ -0,0 +1,70 @@
+import itertools
+
+import torch
+import torch._dynamo
+import torch._inductor.config as config
+from benchmark_helper import time_with_torch_timer
+
+
+@torch._dynamo.optimize("inductor", nopython=True)
+def inductor_scatter_add(dst, src, index):
+    return torch.scatter_add(dst, 1, index, src)
+
+
+def torch_scatter_add(dst, src, index):
+    return torch.scatter_add(dst, 1, index, src)
+
+
+def test_total_time(shapes, types):
+    config.cpp.min_chunk_size = 16
```

Comment:
Please explain why you override this. Best we test with the default configuration.

Reply:
Can you share the perf result of integral types with the default setting?

Reply:
After remove the min_chunk_size, [1, 4096] will not active omp multithread, and the perf result is
```
before (cmpxchg):
[1, 4096]; torch.int32; 0.13751625000168133; 0.051710380000713485; 0.13607534999891868; 0.05633760000137045
[1, 65536]; torch.int32; 1.2092015800021727; 0.16124671000170565; 1.210625030000756; 3.0880012700004045
```
```
after (fetch_add):
```
[1, 4096]; torch.int32; 0.1315705800016076; 0.050090559998352546; 0.1371014300002571; 0.05285493999963364
[1, 65536]; torch.int32; 1.2167823700019653; 0.12195810000093843; 1.1814964000018335; 0.8501880000017081
```
It seems the performance gain without multithread is not as high as multithread.

Reply:
> It seems the performance gain without multithread is not as high as multithread.

Thanks. The result still looks good enough.